src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class Arm extends SubsystemBase {
+
+    private final ArmIO arm;
+
+    private final ArmIOInputsAutoLogged armInputs = new ArmIOInputsAutoLogged();
+
+    private boolean upDown;
+
+    public Arm(ArmIO arm) {
+        this.arm = arm;
+    }
+
+    public void armControl(boolean upDown) {
+        this.upDown = upDown;
+        if (upDown) {
+            arm.setMotorPower(4.0, 4.0);
+        } else {
+            arm.setMotorPower(-4.0, -4.0);
```

Comment:
Let's use PID control in the I/O classes rather than setting a raw power. Then we can control to a target position